corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -203,7 +203,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
                     if (options.target.pendingAnswer &&
                             options.target.pendingAnswer() !== constants.NO_PENDING_ANSWER) {
                         // There is a request in progress
```

Comment:
change comments-- looks like this adds another scenerio

Reply:
In the case that the entry is for a file `answersEqual` doesn't work for us, but there's still a request in progress. We're just handling the pending change in a slightly different way.